pages/api/games/[id].js

```diff
@@ -7,13 +7,21 @@ export default async function handler(request, response) {
 
   if (request.method === "GET") {
     const game = await Game.findById(id);
-   if (!game) {
+    if (!game) {
       return response.status(404).json({ message: "Game not found" });
     }
     return response.status(200).json(game);
-  } if (request.method === "DELETE") {
+  }
+
+  if (request.method === "DELETE") {
     const updatedGame = request.body;
     await Game.findByIdAndDelete(id, updatedGame);
-    response.status(200).json({ status: "Game is successfully deleted."})
+    response.status(200).json({ status: "Game is successfully deleted." });
```

Comment:
I would suggest to add the "early return"s in each case, to prevent checking all other cases afterwards.